jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -226,6 +226,24 @@ def update_display(self, viewer, x, y):
                         (ImvizImageView, CubevizImageView, MosvizImageView, MosvizProfile2DView)):
             self._image_viewer_update(viewer, x, y)
 
+    def _image_shape_inds(self, image):
+        if image.ndim == 3:
+            # cubeviz case
+            return (0, 1)
```

Comment:
Would prefer you keep the `ix_shape` and `iy_shape` assignments here instead of tuples without any explanation.

Reply:
clarified with inline comments